packages/@o3r/schematics/src/utility/wrapper.ts

```diff
@@ -47,16 +48,20 @@ For more details and instructions on how to change these settings, see https://g
 
       if (isReplyPositive) {
         const version = JSON.parse(readFileSync(path.join(__dirname, '..', '..', 'package.json'), 'utf-8')).version;
-        context.addTask(
-          new NodePackageNgAddTask(
-            '@o3r/telemetry',
-            {
-              dependencyType: NodeDependencyType.Dev,
-              version
+        const installTelemetryRule = setupDependencies({
```

Comment:
To reduce the number of install, this function should be called only once per schematics.
Maybe you can align with what it is done in the @o3r/core schematics' helpers